hw/bsp/nordic_pca10056/src/arch/cortex_m4/gcc_startup_nrf52840.s

```diff
@@ -132,6 +132,7 @@ __isr_vector:
     .long   SPIM3_IRQHandler
     .long   0                           /*Reserved */
     .long   PWM3_IRQHandler
+    .long   0                           /*Reserved */

```

Comment:
nrf52840 nvic supports 48 vectors so there are 2 entries missing here, not one
also SPIM3 should be moved to last vector